### DISCUSSION TITLE
FlareSolverr LXC script error

### DISCUSSION THREAD

**Discussion by kokobadu** [asylumexp/Proxmox#74] [Q&A]
While executing FlareSolverr LXC script all the way in the end I get error. It makes no differance with default settings or not.
```
 ✓ Cleaned
 |400 Parameter verification failed.rces   
memory: unable to hotplug memory: trying to change memory cgroup values: container not running

pct set <vmid> [OPTIONS]

[ERROR] in line 75: exit code 0: while executing command pct set $CTID -memory 512
```

**Comment by asylumexp** [comment]
Are you able to give some details about the host you have Proxmox on? I haven't been able to replicate this but has been seen by someone else (#67)

**Comment by kokobadu** [comment]
Many thanks for looking at it!
Proxmox 8.2.4 on RPI5 with 8GB
Operating System: Debian GNU/Linux 12 (bookworm)  
Kernel: Linux 6.6.31+rpt-rpi-2712
Architecture: arm64
1 VM, 3 CT. Error appears when all vm and ct are down as well.

**Reply by asylumexp** [reply]
`apt install debsums && debsums -s`

can you run this on both your host and the LXC and post the output in two txt files?

**Reply by kokobadu** [reply]
I've installed debsums and on my host I get this (after significant amount of time / few minutes):
```
root@raspberrypi5:~# debsums -s
debsums: changed file /usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js (from proxmox-widget-toolkit package)
root@raspberrypi5:~# 
```
and on the FlareSolverr LXC I get this:
```
root@flaresolverr:~# debsums -s                                                                                                            
debsums: missing file /usr/lib/python3.11/EXTERNALLY-MANAGED (from libpython3.11-stdlib:arm64 package)
root@flaresolverr:~#
```

**Reply by asylumexp** [reply]
What type of RPI are you using?

**Reply by kokobadu** [reply]
RPI5 with 8 GB of ram.

**Reply by asylumexp** [reply]
Okay just added a line to start the container, can you run the script as verbose and tell me what it says at the end?

**Reply by kokobadu** [reply]
Thank you. That is the outcome:
```
 ✓ Cleaned
 /CT 103 already runningNormal Resources   
 -
[ERROR] in line 75: exit code 0: while executing command pct start $CTID
```

**Reply by kokobadu** [reply]
Anything I can do to clarify it even more? Would be happy to do it.
Still getting the same outcome as before.

**Reply by asylumexp** [reply]
Yeah apologies for no reply on this, I can't replicate it on my side, and I'm assuming that its a rpi5 issue, does the container itself have any issues in working?  If it doesn't then all you need to do is just manually set the memory to 512mb and to 1 core, but if it doesnt work then just tell me and I'll continue to try fix it.
